sdk/src/androidTest/java/com/hcaptcha/sdk/HCaptchaTestHtml.java

```diff
@@ -0,0 +1,46 @@
+package com.hcaptcha.sdk;
+
+class HCaptchaTestHtml implements IHCaptchaHtmlProvider {
+
+    @Override
+    public String getHtml() {
+        return "<html>\n"
```

Comment:
Can't we just do:
```
return """
...
"""
```

Reply:
The `\n` should be unnecessary as well.

Reply:
AFAIK it is supported only for Java 13 (https://blogs.oracle.com/javamagazine/post/text-blocks-come-to-java), while we use Java 8. To be honest I'm not quite sure that even all Java 8 features supported for Android API 16

Reply:
@e271828- regarding `\n`  we have some useful comments inside for HTML which I think make sense to keep. But once we removed all `\n` HTML becomes partially commented and invalid. I prefer to keep `\n`